ara/exec/parser/manifest_parser.cpp

```diff
@@ -99,9 +134,15 @@ const std::vector<std::string> kAsVector{kApplicationManifest,
 
 namespace MMJsonKeys {
 const std::string kMachineManifest = "Machine_manifest";
-// TODO add rest of the keys
+const std::string kMachineManifestId = "Machine_manifest_id";
+const std::string kModeDeclarationGroup = "Mode_declaration_group";
+const std::string kFunctionGroupName = "Function_group_name";
+const std::string kModeDeclarations = "Mode_declarations";
+const std::string kMode = "Mode";
 
-const std::vector<std::string> kAsVector{kMachineManifest /* TODO add rest of the keys */};
+const std::vector<std::string> kAsVector{kMachineManifest,      kMachineManifestId,
```

Comment:
Kazda wartosc w nowej linii jest bardziej czytelna niz po dwie. Szczegolnie ze odpowiednik dla aplikacji jest pojedynczo w linii.

Reply:
To clang-format tak uklada ;)